axelrod/strategies/dbs.py

```diff
@@ -0,0 +1,398 @@
+from axelrod.actions import Actions, Action
+from axelrod.player import Player
+
+
+C, D = Actions.C, Actions.D
+
+
+class DBS(Player):
+    """
+    Desired Belief Strategy as described in [Au2006]_
+    http://www.cs.utexas.edu/%7Echiu/papers/Au06NoisyIPD.pdf
+
+    A strategy that learns the opponent's strategy, and use symbolic 
+    noise detection for detecting whether anomalies in player’s behavior
+    are deliberate or accidental, hence increasing performance in noisy 
+    tournaments.  
+
+    From the learned opponent's strategy, a tree search is used to
+    choose the best move
+
+    Default values for the parameters are the suggested values in the
+    article. When noise increases you can try to diminish 
+    violation_threshold and rejection_threshold
+
+    Parameters:
```

Comment:
The docstring format for these parameters isn't quite correct. We use the numpy standard described at https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

e.g. The first parameter description should be:
```
    discount_factor: float
        used when computing discounted frequencies to learn opponent's strategy
        Must be between 0 and 1. The default is 0.75
```